### PR TITLE
Tests #38

### PR DESCRIPTION
**Pull request by what-the-hub**
**DRAFT PR**

**Refs:**

[Add validators/helper.ts tests](https://github.com/what-the-hub/vue-game/issues/38)

**Proposed Changes:**

- Init and configure **Jest**
- Add test to `validatePassword.ts`

**Additional Information:**

- Test coverage for `validatePassword.ts` 100%
- Add returned type to `helpers.ts`
- Add scripts to ` package.json ` for runing tests and checking coverage

**Screenshots:**

image

### CODE REVIEW

**Review by Kregopaulgue — APPROVED** [review]

File: src/validators/__tests__/helpers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { validatePassword } from '../helpers'
+
+describe('test validatePassword function', () => {
+  it('should return TRUE for validatePassword("Password123")', function () {
```
Comment (silchencko) [inline src/validators/__tests__/helpers.spec.ts:4]:
```suggestion
  it('should return TRUE if password contains latin characters and numbers', function () {
```

File: src/validators/__tests__/helpers.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    expect(validatePassword('Password123')).toBe(true)
+  })
+  it('should return FALSE for validatePassword("Пароль123")', function () {
```
Comment (silchencko) [inline src/validators/__tests__/helpers.spec.ts:7]: 'should return FALSE if password contains not latin characters'

File: src/validators/__tests__/helpers.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    expect(validatePassword('Пароль123')).toBe(false)
+  })
+  it('should return FALSE for validatePassword("---------")', function () {
```
Comment (silchencko) [inline src/validators/__tests__/helpers.spec.ts:10]: 'should return FALSE if...

File: src/validators/__tests__/helpers.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    expect(validatePassword('---------')).toBe(false)
+  })
+  it('should return FALSE for validatePassword("Psdf24")', function () {
```
Comment (silchencko) [inline src/validators/__tests__/helpers.spec.ts:13]: 'should return FALSE if...

File: src/validators/__tests__/helpers.spec.ts (hunk 1 above)
Comment (what-the-hub) [reply src/validators/__tests__/helpers.spec.ts:4]: Thanks! I just edit code according to this comment in next [PR](https://github.com/what-the-hub/vue-game/pull/43/commits/81c59eeda875f83180b46d58801bd36fc53c68c9)

File: src/validators/__tests__/helpers.spec.ts (hunk 2 above)
Comment (what-the-hub) [reply src/validators/__tests__/helpers.spec.ts:7]: Thanks! I just edit code according to this comment in next [PR](https://github.com/what-the-hub/vue-game/pull/43/commits/81c59eeda875f83180b46d58801bd36fc53c68c9)

File: src/validators/__tests__/helpers.spec.ts (hunk 3 above)
Comment (what-the-hub) [reply src/validators/__tests__/helpers.spec.ts:10]: Thanks! I just edit code according to this comment in next [PR](https://github.com/what-the-hub/vue-game/pull/43/commits/81c59eeda875f83180b46d58801bd36fc53c68c9)

File: src/validators/__tests__/helpers.spec.ts (hunk 4 above)
Comment (what-the-hub) [reply src/validators/__tests__/helpers.spec.ts:13]: Thanks! I just edit code according to this comment in next [PR](https://github.com/what-the-hub/vue-game/pull/43/commits/81c59eeda875f83180b46d58801bd36fc53c68c9)

**Pull request merged**